adapters/base-quarkus/pom.xml

```diff
@@ -0,0 +1,224 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
IMHO we should add the Eclipse copyright header as we do in the other POM files?